krun/tests/test_util.py

```diff
@@ -2,12 +2,15 @@
                        log_and_mail, fatal,
                        check_and_parse_execution_results,
                        run_shell_cmd,
-                       ExecutionFailed)
+                       ExecutionFailed, get_session_info)
```

Comment:
Is it worth going so far as to rename things like `ExecutionFailed` to be consistent with the new terminology?